bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types.go

```diff
@@ -296,6 +296,10 @@ type User struct {
 	// SSHAuthorizedKeys specifies a list of ssh authorized keys for the user
 	// +optional
 	SSHAuthorizedKeys []string `json:"sshAuthorizedKeys,omitempty"`
+
+	// HashedPasswd specifies a hashed password for the already exist user
```

Comment:
Just to emphasize it a bit more.

```suggestion
	// HashedPasswd specifies a hashed password for the user.
	// NOTE: HashedPasswd can be used for existing and new users.
```

+

```go
	// Passwd specifies a hashed password for the user.
	// NOTE: Passwd can be only used for new users.
	// +optional
	Passwd *string `json:"passwd,omitempty"`
```